src/nvim/shada.c

```diff
@@ -10,9 +5,6 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <errno.h>
-#ifdef HAVE_UNISTD_H
-# include <unistd.h>
-#endif
```

Comment:
:+1: I just reverted that change. It seems excessive to include a shared header such as `os/os_defs.h` just for one system header.